### PR TITLE
Drone devices removal

### PR DESCRIPTION
**Pull request by cuberound**
# About the pull request

removes unused shit

# Explain why it's good for the game

cleaner code

# Testing Photographs and Procedure
<details>
<summary>Screenshots & Videos</summary>

Put screenshots and videos here with an empty line between the screenshots and the `<details>` tags.

</details>

# Changelog
:cl:
del:/obj/item/device/gripper and /obj/item/device/matter_decompiler removed
/:cl:

### CODE REVIEW

**Comment by TheManWithNoHands** [comment]
I still member the little drone from comtecs. Sad to see its left overs going to pass on.
Sadge.

**Review by Nanu308 — CHANGES_REQUESTED** [review]
Add in a prop subtype that uses the sprite for it, we mappers like to keep thing looking the same.

File: code/game/objects/prop.dm [hunk 1]
```diff
@@ -169,6 +169,18 @@
 	icon_state = "game_kit"
 	icon = 'icons/obj/items/items.dmi'
 
+/obj/item/prop/griper
```
Comment (Zonespace27) [inline code/game/objects/prop.dm:172]: misspelled.

Comment (cuberound) [reply code/game/objects/prop.dm]: right, fixed now

**Review by realforest2001 — APPROVED** [review]

**Review by Nanu308 — COMMENTED** [review]
mapping lgtm

**Pull request merged**